plugins/inputs/opcua/README.md

```diff
@@ -2,8 +2,9 @@
 
 The `opcua` plugin retrieves data from OPC UA Server devices.
 
-Telegraf minimum version: Telegraf 1.16
-Plugin minimum tested version: 1.16
+⭐ Telegraf v1.25.0
+🏷️ iot, messaging
+💻 all
```

Comment:
Hmmm shouldn't the version be `v1.16.0` instead of `v1.25.0`? Furthermore, I would remove the `messaging` tag as we usually use this for message brokers.